pkgs/tools/X11/xssstate/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchgit
+, libX11
+, libXScrnSaver
+}:
+stdenv.mkDerivation rec {
+  name = "xssstate";
+  #
+  # Use the date of the last commit, since there were bug fixes after the 1.1
+  # release.
+  #
+  version = "20220924";
```

Comment:
```suggestion
  version = "2022-09-24";
```

Reply:
Fixed as of 957a4628244278c8e61bde1cfe7eeab84d018386.